apps/src/componentLibrary/slider/Slider.tsx

```diff
@@ -1,47 +1,96 @@
 import classnames from 'classnames';
-import React, {ChangeEvent, HTMLAttributes} from 'react';
+import React, {
+  ChangeEvent,
+  HTMLAttributes,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
 
-import {componentSizeToBodyTextSizeMap} from '@cdo/apps/componentLibrary/common/constants';
-import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
-import Typography from '@cdo/apps/componentLibrary/typography';
+import {Button, ButtonProps} from '@cdo/apps/componentLibrary/button';
 
 import moduleStyles from './slider.module.scss';
 
 export interface SliderProps extends HTMLAttributes<HTMLInputElement> {
-  /** Slider onChange handler*/
+  /** Slider onChange handler
+   * @param {ChangeEvent<HTMLInputElement>} event - The change event triggered by the slider. */
   onChange: (event: ChangeEvent<HTMLInputElement>) => void;
   /** The name attribute specifies the name of an input element.
-     The name attribute is used to reference elements in a JavaScript,
-     or to reference form data after a form is submitted.
-     Note: Only form elements with a name attribute will have their values passed when submitting a form. */
+   * The name attribute is used to reference elements in JavaScript,
+   * or to reference form data after a form is submitted.
+   * Note: Only form elements with a name attribute will have their values passed when submitting a form. */
   name: string;
   /** The value attribute specifies the value of an input element. */
-  value?: string;
+  value: number | string;
```

Comment:
What does a string value mean here? I see that min max and step are all numbers

Reply:
That's just a native types of input's value. I will check if we can leave just `number` type here and update if yes

Reply:
Double checked, this is caused bu onChange handler event type (`ChangeEvent<HTMLInputElement>`) which declares that value can be number|string.